### DISCUSSION TITLE
Correctly for using multiple output profiles with OriginMapStore

### DISCUSSION THREAD

**Discussion by pisken** [OvenMediaLabs/OvenMediaEngine#1452] [Help]
Hi OME,

Can OME help me use multiple output profiles, context:
+ 1 Origin Mode: multiple output profiles, example: <stream_name>_480, <stream_name>_720
+ n Edge Mode: 
       + case 1: Connect to Origin by direct *Origins* is working
       + case 2: Connect to Origin by *OriginMapStore* not working for multiple profiles, but still work for single profile (not transcode)

Currently, in case 2, I was checked in redis with result, have two keys:
+ `<app>/<stream_name>`: value is ovt url correct.
+ `<app>/<stream_name>_720`: value is `/<app>/<stream_name>_720`. So not working.

I want to woking in case 2 for reduce OVT open connection for get old stream that stopped

**Comment by getroot** [comment]
I moved this to issue https://github.com/AirenSoft/OvenMediaEngine/issues/1453. This is a design error.
